src/python/CMakeLists.txt

```diff
@@ -129,10 +129,13 @@ if(PYTHONINTERP_FOUND)
       set(GUDHI_PYTHON_EXTRA_COMPILE_ARGS "${GUDHI_PYTHON_EXTRA_COMPILE_ARGS}'-DDEBUG_TRACES', ")
     endif()
 
+    set(GUDHI_USE_EIGEN3 "False")
```

Comment:
why not an `else` branch?

Reply:
Ok, it may make you think that no Eigen is the default behaviour. Else branch on ba3be8d1